### PR TITLE
[TR] PIM-4902: Fix bug with channel removal and product save

### PR DESCRIPTION
**Pull request by willy-ahva**
| Q                  | A
| ------------------ | ---
| Migration script?  | n
| Behats             | running
| Specs & Static     | ok
| Changelog updated  |

### CODE REVIEW

File: features/product/save_product_when_channel_has_been_removed.feature [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      | artichoke | vegetable | Yummy !                  |
+    And I am logged in as "Julia"
+
+  Scenario: Successfully delete a channel and continue to work with products which used the channel
+    Given I am on the channels page
+    And I click on the "Delete" action of the row which contains "tablet"
+    And I confirm the deletion
+    Given I am on the "artichoke" product page
+    And I save the product
+    Then I should see a flash message "Product successfully updated."
```
Comment (sumbobyboys) [inline features/product/save_product_when_channel_has_been_removed.feature]: Please don't use flash messages, they are not used anymore

Comment (jjanvier) [inline features/product/save_product_when_channel_has_been_removed.feature]: +1, modify the product and check that the product has been updated

Comment (willy-ahva) [inline features/product/save_product_when_channel_has_been_removed.feature]: ok guys, thanks !

**Comment by ClementGautier** [comment]
GTM after behat updates

**Comment by jjanvier** [comment]
GTM once fixed

**Pull request merged**